tutorials/ceph-cluster/index.mdx

```diff
@@ -45,106 +45,122 @@ dates:
   ```
 
-3. Add the Ceph repository to the apt package manager:
+3. Type the following command to add the Ceph repository to the apt package manager:
 
   ```
   echo deb https://eu.ceph.com/debian-octopus/ $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/ceph.list
   ```
 
-4. Update the apt package manager to include Ceph's repository. Then install `ceph-deploy`:
-
+4. Type the following command to update the apt package manager to include Ceph's repository:
   ```
   sudo apt update
+  ```
+5. Type the following command to install `ceph-deploy`:
+
+  ```
   sudo apt install ceph-deploy
   ```
 
 ## Creating a ceph-deploy user
 
-To be able to install software and configuration files on the storage nodes, `ceph-deploy` needs access to a user that has passwordless sudo privileges.
+`ceph-deploy` needs access to a user that has passwordless sudo privileges to be able to install software and configuration files on the storage nodes.
 Ceph deploy will be able to connect as any user, including the `root` user, however this is not recommended. To simplify the software configuration, it is recommended to use the same username on all Ceph nodes in the cluster.
 
-<Message type="tip">
+<Message type="important">
 
-Do not use `ceph` as login for your user, as it is reserved for Ceph-daemons. The installation of Ceph will fail if a `ceph` user exists on a machine.
+ Do not use `ceph` as login for your user, as it is reserved for Ceph-daemons. The installation of Ceph will fail if a `ceph` user exists on a machine.
 
 </Message>
 
-1. Connect to your Ceph node using SSH:
+1. Type the following command to connect to your Ceph node using SSH:
 
   ```
   ssh user@ceph-node
   ```
 
-2. Create a user called `ceph-deploy` (You can rename it to your own preferences if required)
+2. Type the following command to create a user called `ceph-deploy`: 
 
   ```
   sudo useradd -d /home/ceph-deploy -m ceph-deploy
   ```
+ 
+  <Message type="note">
 
-3. Configure the password of the `ceph-deploy` user:
+   You can rename the user to your own preferences if required.
+
+  </Message>
+
+3. Type the following command to configure the password of the `ceph-deploy` user:
 
   ```
   sudo passwd ceph-deploy
   ```
 
-4. Add the user to the sudoers configuration so it has sudo rights:
+4. Type the following command to add the user to the sudoers configuration so they have sudo rights:
 
   ```
   echo "ceph-deploy ALL = (root) NOPASSWD:ALL" | sudo tee /etc/sudoers.d/ceph-deploy
   sudo chmod 0440 /etc/sudoers.d/ceph-deploy
   ```
 
-5. Install an NTP client on your Ceph nodes to avoid synchronization issues caused by time-drift:
+5. Type the following command to install an NTP client on your Ceph nodes:
 
   ```
   sudo apt install ntpsec
   ```
 
-6. Install Python, which is required by `ceph-deploy` to deploy the cluster:
+  <Message type="note">
+
+   The installation of an NTP client allows you to avoid synchronization issues caused by time-drift.
```

Comment:
```suggestion
   The installation of an [NTP](https://en.wikipedia.org/wiki/Network_Time_Protocol) client allows you to avoid synchronization issues caused by time-drift.
```